lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -47,28 +113,15 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
     _subscriptions.clear();
     _lastEditingState = null;
 
-    // If focused element is a part of a form, it needs to stay on the DOM
```

Comment:
Looks like this was added as a fix for form autofill, does disabling keep form submit intact?

Reply:
No, I removed form autofill from semantics. It was broken. The non-a11y version is still intact.